config/sources/families/tqma8mpxl.conf

```diff
@@ -0,0 +1,68 @@
+#
+# SPDX-License-Identifier: GPL-2.0
+#
+# Copyright (c) Martin Schmiedel
+#
+# This file is a part of the Armbian Build Framework
+# https://github.com/armbian/build/
+#
+
+declare -g ARCH="arm64"
+declare -g KERNEL_IMAGE_TYPE="Image"
+declare -g OFFSET="32"
+declare -g SERIALCON=ttymxc3
+declare -g IMX_FIRMWARE="firmware-imx-8.12"
+declare -g BOOTENV_FILE="tqma8mpxl.txt"
+
+# ATF related
+# MAP = target_make;target_patchdir;target_files
+declare -g ATF_TARGET_MAP="PLAT=imx8mp IMX_BOOT_UART_BASE=0x30a60000 bl31;;build/imx8mp/release/bl31.bin"
+# If needed use tq-systems repo
+ATFSOURCE='https://github.com/tq-systems/atf'
+ATFBRANCH="branch:TQMa8-imx_5.4.70_2.3.0"
+
+# Firmware related
+
+pre_config_uboot_target()
+{
+	# get the firmware
+	rm -rf ${IMX_FIRMWARE}*
+	wget https://www.nxp.com/lgfiles/NMG/MAD/YOCTO/${IMX_FIRMWARE}.bin -O ${IMX_FIRMWARE}.bin
+	chmod +x ${IMX_FIRMWARE}.bin
+	./${IMX_FIRMWARE}.bin --auto-accept
+
+	# copy needed files for uboot
+	cp ${IMX_FIRMWARE}/firmware/hdmi/cadence/signed_hdmi_imx8m.bin .
```

Comment:
That's only for imx8mq. Not needed for imx8mp

Reply:
Okay. Add `signed_dp_imx8m.bin` as well. i.MX8M is capable of supporting either DP or HDMI. Each mode needs its own firmware file

Reply:
Done.